src/components/Explore/TokenDetails/LoadingTokenDetail.tsx

```diff
@@ -71,6 +77,15 @@ const Space = styled.div<{ heightSize: number }>`
   height: ${({ heightSize }) => `${heightSize}px`};
 `
 
+function Wave() {
+  const theme = useTheme()
+  return (
+    <svg width="416" height="160" xmlns="http://www.w3.org/2000/svg">
+      <path d="M 0 80 Q 104 10, 208 80 T 416 80" stroke={theme.backgroundModule} fill="transparent" strokeWidth="2" />
```

Comment:
I think this should either be backgroundInteractive or backgroundOutline -- because the rest of the loading bubbles are that color animation (in figma it shows some other color not in the theme file so)

Reply:
The inspector on Figma shows it's using background module on my end